pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -339,6 +356,20 @@ func (r *ReconcileNetworkAddonsConfig) applyObjects(networkAddonsConfig *opv1alp
 	return nil
 }
 
+// Delete removed objects
+func (r *ReconcileNetworkAddonsConfig) deleteObjects(networkAddonsConfig *opv1alpha1.NetworkAddonsConfig, objs []*unstructured.Unstructured) error {
+	for _, obj := range objs {
+		// Apply all objects on apiserver
+		if err := apply.DeleteObject(context.TODO(), r.client, obj); err != nil {
+			log.Printf("could not apply (%s) %s/%s: %v", obj.GroupVersionKind(), obj.GetNamespace(), obj.GetName(), err)
+			err = errors.Wrapf(err, "could not apply (%s) %s/%s", obj.GroupVersionKind(), obj.GetNamespace(), obj.GetName())
```

Comment:
Again, this is not about apply but delete. Also, we should be graceful when deleting, if the object is not found, we don't need to fail. On the other hand, if we fail to remove an existing object, we should raise an error. Please handle it in the DeleteObject funnction.

Reply:
the logic you said is already there according what i checked now, see DeleteObject, in case i am not wrong
but /s/apply/delete/g/
thanks